mods/rv/rules/default-structures.yaml

```diff
@@ -145,6 +146,9 @@
 	Voiced:
 		VoiceSet: ConstructionYardVoice
 		Volume: 0.8
+	ProvidesPrerequisite@AIantiRush:
+		RequiresCondition: !ai-control
+		Prerequisite: AIantiRush ## Hack: AI Building control. AI build refinary only when it has a combat producing line.
```

Comment:
This doesn't sound right. If ConYard provides these, then the AI will always have this prerequisite when it can also build a refinery.

Reply:
The condition "!ai-control" make the AI player cannot have this prerequisite before build either war factory or barrack.

Reply:
War factory and barrack directly provide the prerequisite.

Reply:
If player is not AI, this prerequisite will provide directly by conyard